utils.py

```diff
@@ -88,8 +89,12 @@ def compute_ram_memory_stats(byte_scaler =1024 ** 3):
 
 
 def compute_memory_fragmentation():
-	snapshot = torch.cuda.memory_snapshot()
-	return sum(b['allocated_size'] for b in snapshot) / sum(b['total_size'] for b in snapshot)
+    lib = ctypes.cdll.LoadLibrary(None)
```

Comment:
if you want to compute device free/total memory stats, please introduce a new function, do not hijack an existing one

Reply:
answer abowe